src/test/java/com/dropwizard/example/h2/resources/PersonResourceTest.java

```diff
@@ -0,0 +1,150 @@
+package com.dropwizard.example.h2.resources;
```

Comment:
Ok, here you already have JUnit5)) Then clean up dependencies and just fix PersonTest.